kuma/static/js/helpfulness.js

```diff
@@ -16,6 +16,12 @@
 
     // create a notification
     function inquire() {
+        // double chck we havn't asked on another page in the last minute
```

Comment:
The `helpfulnessTracker` is set to `now()` plus 7 days. This comment seems incorrect in that case. (Also, multiple typos)